### PR TITLE
Merged clients + login component

### PR DESCRIPTION
**Pull request by mebeborn**

### CODE REVIEW

**Comment by mebeborn** [comment]
Code style

**Comment by mebeborn** [comment]
Fixed tabs

File: client/src/app/app_guards/logged-out.guard.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { AuthService } from '../app_services/auth.service';
+import { Router } from '@angular/router';
+
+@Injectable()
+export class LoggedOutGuard implements CanActivate {
+  constructor(private authService: AuthService, private router: Router) { }
+
+  canActivate(
+    next: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot): Observable<boolean> | Promise<boolean> | boolean {
```
Comment (idyac) [inline client/src/app/app_guards/logged-out.guard.ts]:
```typescript
canActivate(
    next: ActivatedRouteSnapshot,
    state: RouterStateSnapshot
)
```

File: client/src/app/app.module.ts [hunk 2]
```diff
@@ -1,20 +1,45 @@
...
+    HomeComponent,
+    PageNotFoundComponent
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    FormsModule,
+    HttpModule,
+
```
Comment (idyac) [inline client/src/app/app.module.ts]: ?

File: client/src/app/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<div class="main-wrapper">
+	<div class="card" style="width: 27rem;">
+		<p class="card-title">LOGIN TO <span class="font-weight-bold">HRMS</span></p>
+		<div class="card-body">
```
Comment (idyac) [inline client/src/app/home/home.component.html]: format it to 2 spaces

**Pull request merged**